### PR TITLE
Update Recruiting page.md

### PR DESCRIPTION
**Pull request by HilaryClarke**
Stage 1 of many updates/additions to come.

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks! @HilaryClarke Anyone from HR that should review (e.g. Natalie or Aneal) before merging?

**Comment by HilaryClarke** [comment]
Yes, let's have Natalie review. Thank you!

**Comment by jasonblais** [comment]
@natalie-hub Pinging you to help with review of this PR, thank you!

@justinegeffen pinging you for editor review (grammar/structure)

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
Due to local laws, there may be restrictions on which types of roles we can hire in each country/region.
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
Preparing to hire an employee in a new state takes time, and we may be subject to new regulations, taxes, and reporting obligations. If a candidate is progressing in the interview process and would be living and working from a U.S. state not listed above, the Recruiter will DM @tim.quock with the state and the title of the position the candidate may fill at least three weeks prior to an anticipated offer. 
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
Staff agreements are conditioned upon the country/region of residence noted in a staff member's contract. Location change requests should be discussed with HR and your MLT member. Staff must receive written approval from the Company prior to a country/region relocation.
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
Lever is our Applicant Tracking System (ATS). It is where our Recruiters, Recruiting Coordinators, and Hiring Managers organize open roles, track a candidate’s progress throughout the recruiting process, collect feedback, and record data. It is important that all candidates, interviews, and feedback are recorded accurately in Lever because all of our data and insights are pulled from Lever. Without accurate data in Lever, we aren’t able to iterate on the process to deliver a better hiring experience for candidates, hiring managers, and the business overall. 
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
1. Hire/No Hire feedback from everyone on the interview loop.
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
2. Attachments of any work artifacts produced during the interview loop.
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
3. Any open questions from interviewers that have not been covered.
```

File: contributors/join-us/recruiting.md
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
4. Notes on any backchannel references completed. For any confidential feedback please use SECRET NOTE with mention to the AA and leave out the name of the source.
```

**Review by justinegeffen — COMMENTED** [review]
Few small changes but otherwise looks good, thanks @HilaryClarke!

**Comment by HilaryClarke** [comment]
Thanks so much for the edits @justinegeffen, they look great!

**Review by justinegeffen — DISMISSED** [review]
Thanks, @HilaryClarke, this is great. :)

**Review by natalie-hub — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]
This looks good to me! @HilaryClarke, is this ready to merge? :)

**Comment by HilaryClarke** [comment]
@justinegeffen Thanks Justine, ready to merge!

**Pull request merged**
